service-stubs/identity/org.wso2.carbon.identity.application.mgt.stub/src/main/resources/IdentityApplicationManagementService.wsdl

```diff
@@ -419,9 +419,13 @@
                     <xs:element minOccurs="0" name="inboundAuthType" nillable="true" type="xs:string"/>
                     <xs:element minOccurs="0" name="inboundConfigType" nillable="true" type="xs:string"/>
                     <xs:element minOccurs="0" name="inboundConfiguration" nillable="true" type="xs:string"/>
+                    <xs:element minOccurs="0" name="inboundConfigurationProtocol" nillable="true" type="ax2169:InboundConfigurationProtocol"/>
```

Comment:
Test the export through the UI

Reply:
Import/export function still works as intended from the console. But the Import function only supports exported files from  the console or from the ```/export``` API. Does not support the exported files from the ```/exportFile``` API.